test/Interop/Cxx/class/destructor-correct-abi-irgen.swift

```diff
@@ -0,0 +1,13 @@
+// RUN: %swift -I %S/Inputs -enable-cxx-interop -emit-ir %s | %FileCheck %s
```

Comment:
To keep file names related to this test consistent, could you rename the file to `destructors-...`?